doc/data/messages/c/consider-using-generator/details.rst

```diff
@@ -0,0 +1,5 @@
+Removing ``[]`` inside calls that can use containers or generator indistinctly should be considered for performance reasons since a generator will have an upfront cost to pay. The performance will be better if
+you are working with really long lists or sets and depending on the interpreter: you should consider
+your dataset and your environment.
```

Comment:
```suggestion
you are working with long lists or sets.
```

"Depending on you're interpreter" doesn't really add much imo. Documentation like this is mostly meant for beginners and without clear directions about what to then think about when they are thinking about their interpreter it doesn't add much and just confuses them. Length of the iterator is a specific tip so that works well.